src/pages/Dashboard/Dashboard.tsx

```diff
@@ -22,12 +30,48 @@ export const Dashboard: React.FC = (): React.ReactElement => {
           dataKeyXAxis="name"
           dataKeyArea="Applicants"
         />
-        <DashCard title="Current Vacancies" content="9" />
-        <DashCard title="Current Applicants" content="40" />
-        <DashCard title="Interviews Today" content="4" />
-        <DashCard title="Callbacks Required" content="10" />
+        <DashCard
+          title="Active Vacancies"
+          content="9"
+          icon={
+            <FontAwesomeIcon className="dashboard-icon" icon={faNewspaper} />
```

Comment:
Same as above ^^ 

Here you should probably just be passing the icon needed

<DashCard title="Active Vacancies" content="9"  icon={faNewspaper} />